source/operating-a-service/incident-playbook.html.md

```diff
@@ -0,0 +1,91 @@
+# Incident playbook
+
+## Once an incident happens
+
+### 1. Form an incident response team
+
+Self-organise to appoint:
+
+1.  the **Incident comms lead**
+    (responsible for communicating and alerts to users and stakeholders)
+2.  the **Incident tech lead**
+    (responsible for technical direction and communicating to the comms lead)
+3. the **Incident support lead**, when the incident involves a public-facing service
+    (responsible for monitoring Zendesk and alerting the team to any changes in the queues and severity of experience of users).
+
+### 2. Triage the incident (all incident leads)
+
+The incident leads should [triage the incident](/operating-a-service/how-to-categorise-technical-incidents.html) (P1, P2, P3).
+
+### 3. Create an incident Slack channel and inform the stakeholders (comms lead)
+
+1. Initiate the Slack IncidentBot by typing `/incident` in the message box on #twd_git_bat channel or the #tra_digital channel (as appropriate), and hit Enter.
+2. Complete the details in the IncidentBot template, and press Enter, which will automatically create a dedicated slack channel for the incident.
+3. Determine who needs to be contacted, based on the incident priority and affected services, using [the incident contact lists](https://docs.google.com/document/d/1E3sL-Om_NPHWHdYdLykVuiWux_6AmR5pn5KjIkQYHaI/edit#bookmark=id.djbosiwhbjjy).
+4. @tag the appropriate people (from the tables linked below) in the dedicated channel:
+
+### 4. Provide a service update to users outside DfE (comms lead)
+
+The Teacher Services team maintains a publicly available [service status dashboard](https://teacher-services-status.education.gov.uk/). During an incident, the comms lead needs to explain what’s happening to users outside DfE. The comms lead will need a GitHub account to do this, or delegate updates to a colleague who has one.
+
+The updates are managed via GitHub Actions and issues on the [teacher-services-upptime repository on GitHub](https://github.com/DFE-Digital/teacher-services-upptime). If a service’s automatic health check is failing continuously, an issue will be created within 5 minutes of the failure occurring and the dashboard will start reporting a service issue.
+
+To update the dashboard:
+
+1. Navigate to the appropriate incident issue on the [GitHub issues page](https://github.com/DFE-Digital/teacher-services-upptime/issues)
+2. Add a comment to the issue
+
+### 5. Start the incident report (any incident lead)
```

Comment:
This is also in the Slack bot instructions — do we want to duplicate out here?

Reply:
I guess I see the bot as a progressive enhancement over these instructions, so I'd probably suggest leaving them in both, but not sure whether that gels with how others see it.

Reply:
Fair — let's keep it here!